### ISSUE TITLE
uc2otutab.py not producing otu table in Alfa_DADA2_vsearch.sh

### ISSUE THREAD

**Issue by lplough** [tobiasgf/lulu#9]
Hi,
I followed the very excellent(!) pipeline and scripts you provided as part of LULU to go from DADA2 ASV output to inputs for vsearch clustering using 'extrSamDADA2', and ran the Alfa_DADA2_vsearch script. All works fine until the uc2otutab.py script is called, which has biopython fasta as a dependency. In order to install biopython in my virtual env. I had to use python3.6 which then produces print errors for scripts from the other python dependencies (e.g. click , uc) which must have been written for python2.X. After altering scripts to print in python3 (adding parentheses; not sure if that was the best route?), the Alfa_DADA2_vsearch.sh completes and the log shows no errors, and makes .uc files and .centroid files, but the .otu tables for each level of clustering are empty. It's not clear if my .uc (or even fasta) input files are off or if the uc2otutab.py script is not working due to a python version issue?  

I would be happy to simply run vsearch directly on my files at some defined level of clustering and get otu ouput, i.e, 

vsearch --cluster_size input.fasta --id 0.97 --relabel OTU --biomout otutable.biom

as you suggested here: https://github.com/torognes/vsearch/issues/166

but Im not sure what file input to use? I have not used vsearch much before.   I understand I should not use a dereplicated fasta- does the extrSamDADA2 script produce that file? Can I run usearch_global on the centroid and uc files with the --biomout argument? 

Basically, the uc2otutab.py script is not working on my input files, so I am looking for an alternative way to cluster and output an otutable with vsearch. Ill attach my dummy sample fasta files that were input to the Alfa_DADA2_vsearch.sh script (two identical samples from a reduced dataset) to see if they are perhaps part of the issue? 
[Fastas.zip](https://github.com/tobiasgf/lulu/files/4793423/Fastas.zip)
 
Thanks for any guidance you can provide!

Louis
